sdk/communication/azure-communication-chat/build.gradle

```diff
@@ -0,0 +1,27 @@
+ext.publishName = "Microsoft Azure Android Client Library for Communication Chat Services"
+description = "This package contains the Android client library common code for Microsoft Azure Communication Chat Service."
+version = "1.0.0-beta.2"
+ext.versionCode = 1
+
+android {
+    defaultConfig {
+        versionCode project.versionCode
+        versionName project.version
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
+    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-xml:$jacksonVersion"
+    implementation "com.jakewharton.threetenabp:threetenabp:$threeTenAbpVersion"
+    implementation "com.squareup.okhttp3:okhttp:$okHttpVersion"
+    implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
+    implementation project(":sdk:core:azure-core")
+    testImplementation "junit:junit:$jUnitVersion"
+    testImplementation "com.squareup.okhttp3:mockwebserver:$okHttpVersion"
+    androidTestImplementation "androidx.test.ext:junit:1.1.2"
+    androidTestImplementation "androidx.test.espresso:espresso-core:3.3.0"
```

Comment:
We should avoid using hardcoded versions in `build.gradle` files.

Reply:
Yes. But this also happened in Storage. I was thinking to make another PR to address this later.